machine.go

```diff
@@ -185,16 +185,17 @@ type image struct {
 }
 
 type Machine struct {
-	arch     Arch
-	backend  backend
-	mounts   []mountPoint
-	count    int
-	images   []image
-	memory   int
-	numcpus  int
-	showBoot bool
-	quiet    bool
-	Environ  []string
+	arch       Arch
+	backend    backend
+	mounts     []mountPoint
+	count      int
+	images     []image
+	memory     int
+	numcpus    int
+	sectorSize int
```

Comment:
This changes are not related

Reply:
I think this is actually just from gofmt.

Reply:
'sectorSize' is longer than other struct members, so without this change, I'm hitting the linter error:

'''
machine.go:188:1: File is not properly formatted (gofmt)
	arch	  Arch
'''